shared/src/packages/trucks/libs/validation-schemas/validation-schema.ts

```diff
@@ -0,0 +1,3 @@
+export { truckCreateRequestBodyValidationSchema } from './truck-create-request-body.validation-schema.js';
+export { truckIdParameterValidationSchema } from './truck-id-parameter.validation-schema.js';
+export { truckUpdateRequestBodyValidationSchema } from './truck-update-request-body.validation-schema.js';
```

Comment:
we are calling them without ValidationSchema inside naming

Reply:
ok